html/user-activation/consumption-sameorigin.html

```diff
@@ -84,25 +84,31 @@
         // Phase switching.
         if (msg.type.endsWith("-loaded")) {
             if (--num_children_to_load == 0)
-                finishLoadPhase();
+                await finishLoadPhase();
         } else if (msg.type.endsWith("-report")) {
             if (--num_children_to_report == 0)
                 finishReportPhase();
         }
     });
+    async function createIframes() {
+        const child1 = document.createElement("iframe");
+        child1.src = "resources/child-one.html";
+        child1.id = "child1";
+        document.body.appendChild(child1);
+        await new Promise((resolve) => (child1.onload = resolve));
+        const childSO = document.createElement("iframe");
+        childSO.id = "child-so";
+        childSO.src = "resources/consumption-sameorigin-child.html";
+        document.body.appendChild(childSO);
+        await new Promise((resolve) => (childSO.onload = resolve));
```

Comment:
Looks like we are still relying on the `load` event behavior @annevk mentioned above...I mean the fact that `childSO`'s `load` is ultimately blocked by "grandchild" frame's `load`,  right?  I find this mix of flattening-vs-not a bit too awkward!

Reply:
Removed this line. There is no need to await for it to load. What matters is just that the first iframe loads in order. 

I've made similar changes to all the other dual iframe tests. I'll have them re-viewed upstream as part of my WebKit patch and will just send them reviewed as part of that.